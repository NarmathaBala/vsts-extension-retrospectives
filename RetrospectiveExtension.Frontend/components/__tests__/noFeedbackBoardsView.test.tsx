import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
import NoFeedbackBoardsView, { NoFeedbackBoardsViewProps } from '../noFeedbackBoardsView';

let mockOnCreateBoardClick = jest.fn(() => { });

const defaultTestProps: NoFeedbackBoardsViewProps = {
  onCreateBoardClick: mockOnCreateBoardClick
}

describe('No Feedback Boards View component', () => {
  it ('renders correctly.', () => {
    const tree = TestRenderer
      .create(<NoFeedbackBoardsView {...defaultTestProps} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});