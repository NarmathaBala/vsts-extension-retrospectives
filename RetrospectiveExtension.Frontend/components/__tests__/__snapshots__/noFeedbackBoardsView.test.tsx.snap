// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No Feedback Boards View component renders correctly. 1`] = `
<div
  className=""
  onKeyDown={[Function]}
  onMouseDown={[Function]}
  onMouseMove={[Function]}
  onScroll={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
>
  <div
    className="no-boards-container"
  >
    <div
      className="no-boards-text"
    >
      Get started with your first Retrospective
    </div>
    <div
      className="no-boards-sub-text"
    >
      Create a new board to start collecting feedback and create new work items.
    </div>
    <button
      className="ms-Button ms-Button--primary create-new-board-button root-40"
      data-is-focusable={true}
      onClick={[Function]}
      onKeyDown={[Function]}
      onKeyPress={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseUp={[Function]}
      type="button"
    >
      <span
        className="ms-Button-flexContainer flexContainer-41"
        data-automationid="splitbuttonprimary"
      >
        <span
          className="ms-Button-textContainer textContainer-42"
        >
          <span
            className="ms-Button-label label-44"
            id="id__0"
          >
            Create Board
          </span>
        </span>
      </span>
    </button>
  </div>
</div>
`;
